### DISCUSSION TITLE
16. Sécuriser l'accès à l'administration

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#45] [Bonnes pratiques] [answered]
@dNicolle  : @florinesueur demande plus d'info sur "C’est quoi le plus en termes d’écoconception ? "

**Comment by dNicolle** [comment, marked as answer]
IDEM pour la fiche 18. Sécuriser les identifiants d'accès à l'administration

**Reply by florinesueur** [reply]
J'ai bien compris que c'était important pour que son site reste son site mais au final, je me rends pas compte ce que ça apporte au niveau de l'emprunte environnementale. C'est pas explicite à mon sens.

**Reply by dNicolle** [reply]
Oui, j'ai bien compris ça comme ça, je vais ajouter des éléments là dessus (plus concrêts j'espère)

**Reply by florinesueur** [reply]
Merci, c'est super chouette :)
